### PR TITLE
Cherry pick #16298: Fix to dropdown width

### PR DESCRIPTION
**Pull request by joschect**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Cherry Picks #16298 into v7.

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by joschect** [comment]
@ilanbd Here is the cherry pick!

File: change/@fluentui-react-2020-12-25-15-17-03-master.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
```
Comment (ecraig12345) [inline change/@fluentui-react-2020-12-25-15-17-03-master.json:2]: You'll need to delete and re-generate these change files.

Comment (joschect) [reply change/@fluentui-react-2020-12-25-15-17-03-master.json]: Done

**Comment by fabricteam** [comment]
## [Perf Analysis](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">7.0 Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/BaseButton-mount_base.html">922</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/BaseButton-mount.html">957</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Breadcrumb-mount_base.html">40576</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Breadcrumb-mount.html">40726</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Checkbox-mount_base.html">1619</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Checkbox-mount.html">1628</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/CheckboxBase-mount_base.html">1338</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/CheckboxBase-mount.html">1347</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/ChoiceGroup-mount_base.html">5196</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/ChoiceGroup-mount.html">5083</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/ComboBox-mount_base.html">933</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/ComboBox-mount.html">930</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/CommandBar-mount_base.html">7514</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/CommandBar-mount.html">7608</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/ContextualMenu-mount_base.html">12775</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/ContextualMenu-mount.html">12754</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/DefaultButton-mount_base.html">1156</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/DefaultButton-mount.html">1203</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/DetailsRow-mount_base.html">3690</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/DetailsRow-mount.html">3745</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/DetailsRowFast-mount_base.html">3724</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/DetailsRowFast-mount.html">3758</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/DetailsRowNoStyles-mount_base.html">3539</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/DetailsRowNoStyles-mount.html">3477</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Dialog-mount_base.html">1523</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Dialog-mount.html">1522</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/DocumentCardTitle-mount_base.html">1738</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/DocumentCardTitle-mount.html">1751</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Dropdown-mount_base.html">2612</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Dropdown-mount.html">2616</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/FocusTrapZone-mount_base.html">1737</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/FocusTrapZone-mount.html">1723</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/FocusZone-mount_base.html">1862</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/FocusZone-mount.html">1849</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/IconButton-mount_base.html">1823</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/IconButton-mount.html">1801</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Label-mount_base.html">327</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Label-mount.html">352</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Layer-mount_base.html">1995</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Layer-mount.html">1990</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Link-mount_base.html">448</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Link-mount.html">441</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/MenuButton-mount_base.html">1520</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/MenuButton-mount.html">1558</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/MessageBar-mount_base.html">1993</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/MessageBar-mount.html">1999</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Nav-mount_base.html">3279</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Nav-mount.html">3325</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/OverflowSet-mount_base.html">1455</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/OverflowSet-mount.html">1464</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Panel-mount_base.html">1476</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Panel-mount.html">1477</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Persona-mount_base.html">873</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Persona-mount.html">831</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Pivot-mount_base.html">1463</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Pivot-mount.html">1458</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/PrimaryButton-mount_base.html">1331</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/PrimaryButton-mount.html">1308</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Rating-mount_base.html">8131</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Rating-mount.html">8007</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/SearchBox-mount_base.html">1288</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/SearchBox-mount.html">1340</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Shimmer-mount_base.html">2662</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Shimmer-mount.html">2740</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Slider-mount_base.html">1527</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Slider-mount.html">1533</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/SpinButton-mount_base.html">4998</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/SpinButton-mount.html">5104</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Spinner-mount_base.html">404</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Spinner-mount.html">397</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/SplitButton-mount_base.html">3248</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/SplitButton-mount.html">3274</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Stack-mount_base.html">521</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Stack-mount.html">531</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/StackWithIntrinsicChildren-mount_base.html">1636</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/StackWithIntrinsicChildren-mount.html">1652</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/StackWithTextChildren-mount_base.html">4878</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/StackWithTextChildren-mount.html">4988</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/SwatchColorPicker-mount_base.html">10678</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/SwatchColorPicker-mount.html">10425</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/TagPicker-mount_base.html">2782</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/TagPicker-mount.html">2844</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/TeachingBubble-mount_base.html">48910</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/TeachingBubble-mount.html">48987</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Text-mount_base.html">450</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Text-mount.html">437</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/TextField-mount_base.html">1435</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/TextField-mount.html">1420</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Toggle-mount_base.html">873</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/Toggle-mount.html">847</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/button-mount_base.html">103</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16959/merge/perf-test/button-mount.html">107</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: change/@fluentui-react-internal-2020-12-25-15-17-03-master.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
```
Comment (ecraig12345) [inline change/@fluentui-react-internal-2020-12-25-15-17-03-master.json:2]: Delete this file

**Review by ecraig12345 — APPROVED** [review]

**Pull request merged**